env.template

```diff
@@ -23,6 +23,7 @@ DB_ROOT_PASSWD=
 REDIS_HOST=127.0.0.1
 REDIS_PORT=6379
 ENABLE_EXPERIMENTAL_REDIS=true
+#REDIS_PASSWORD=admin_redis
```

Comment:
```suggestion
REDIS_PASSWORD=
```

Reply:
Same with this, by default I leaved the docker-compose for the dev team with non password redis, so I leaved this commentes aswel